Avoid adding four new packages that aren't necessary for the manual page
diff --git a/doc/en/conf.py b/doc/en/conf.py
index 2083e72df..144720801 100644
--- a/doc/en/conf.py
+++ b/doc/en/conf.py
@@ -27,17 +26,11 @@ release = ".".join(version.split(".")[:2])
 
 root_doc = "index"
 extensions = [
-    "pygments_pytest",
     "sphinx.ext.autodoc",
     "sphinx.ext.autosummary",
     "sphinx.ext.intersphinx",
     "sphinx.ext.todo",
     "sphinx.ext.viewcode",
-    "sphinx_removed_in",
-    "sphinx_inline_tabs",
-    "sphinxcontrib_trio",
-    "sphinxcontrib.towncrier.ext",  # provides `towncrier-draft-entries` directive
-    "sphinx_issues",  # implements `:issue:`, `:pr:` and other GH-related roles
 ]
 
 # Building PDF docs on readthedocs requires inkscape for svg to pdf
